frontend/src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  api_root: 'http://localhost:3000',
+  api_root: 'http://localhost:3000/v1',
```

Comment:
**nitpick**: I'd rather be using a header, setting value somewhere in configuration, but I don't yet have a sense of where that would go in our HTTP client setup. Maybe a **todo**.

Reply:
https://angular.io/guide/http#adding-and-updating-headers suggests that each call to `.get()` needs to have an `options` object sent along https://angular.io/guide/http#requesting-data-from-a-server

Reply:
Are you okay with us leaving this as a todo? I added a comment in about setting default headers in the HttpClient, to guide future work.